wordpress/wp-content/plugins/require-api-authentication/require-api-authentication.php

```diff
@@ -1,21 +0,0 @@
-<?php
```

Comment:
This plugin was deleted and its logic was moved to main sil-dictionary-webonary plugin for clarity